### DISCUSSION TITLE
AttributeError: 'HTTPClient' object has no attribute 'send_files'

### DISCUSSION THREAD

**Discussion by Deku-nattsu** [Rapptz/discord.py#6646] [General] [answered]
i am getting this error when trying to send local file
await ctx.send(file=picture)
File "/python/lib/python3.9/site-packages/discord/abc.py", line 1032, in send
data = await state.http.send_files(channel.id, files=[file], allowed_mentions=allowed_mentions,
AttributeError: 'HTTPClient' object has no attribute 'send_files'

**Comment by Rapptz** [comment, marked as answer]
This has been fixed.

Note however that using `master` is prone to breakage and should only be installed if you're fine with instability. It's better to use a stable release such as the ones on PyPI like v1.7.

**Reply by Deku-nattsu** [reply]
ohk good to know, thanks
